universal-login-react/src/ui/Migrating/EnterEmail.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {useState} from 'react';
+import {ensureNotFalsy, isProperEmail} from '@unilogin/commons';
+import {WalletService} from '@unilogin/sdk';
+import {useClassFor, classForComponent} from '../utils/classFor';
+import {InputField} from '../commons/InputField';
+import {useInputField} from '../hooks/useInputField';
+import {PrimaryButton} from '../commons/Buttons/PrimaryButton';
+import '../styles/base/enterEmail.sass';
+import '../styles/themes/UniLogin/enterEmailThemeUniLogin.sass';
+import emailIcon from '../assets/icons/e-mail.svg';
+import {SecondaryButton} from '../commons/Buttons/SecondaryButton';
+import Spinner from '../commons/Spinner';
+import {OnboardingStepsWrapper} from '../Onboarding/OnboardingStepsWrapper';
+import {useHistory} from 'react-router';
+
+const emailValidator = {
```

Comment:
We can extract it to separate file e.g. `inputValidators` and get rid of duplications